classes/Birthday.py

```diff
@@ -1,5 +1,29 @@
 from classes.Field import Field
+from datetime import date, datetime
+
+from helpers.error import BirthdayValueError, DateValueError
+
 
 
 class Birthday(Field):
-    pass
+    def __init__(self, birthday):
+        self.birthday_validation(birthday)
+        super().__init__(birthday)
+
+
+    @property
+    def birthday(self):
+        raise AttributeError('This property has no getter')
+
+    @birthday.setter
+    def birthday(self, birthday):
+        self.__init__(birthday)
+
+    def birthday_validation(self, birthday):
+        try:
+            birthday_date = datetime.strptime(birthday, "%d.%m.%Y").date()
+        except ValueError:
+            raise DateValueError
```

Comment:
Please call the function with argument otherwise it will produce an error
in our case, the error will be displayed incorrectly
DateValueError(birthday)